### DISCUSSION TITLE
Cannot download file

### DISCUSSION THREAD

**Discussion by Hi-Hd** [squidowl/halloy#385] [Q&A] [answered]
Sorry for the noob question 
I cannot download a file from a server
<img width="857" alt="image" src="https://github.com/squidowl/halloy/assets/100292944/33c752e7-9b93-4def-b0c1-6faac26f77e3">
no dialog box to accept the file like in clicksandwhistles windows client

My config file

<img width="919" alt="image" src="https://github.com/squidowl/halloy/assets/100292944/d8e0f50e-3e71-4639-970e-850580325844">

**Comment by tarkah** [comment, marked as answer]
In the bottom left corner of Halloy, there's a box icon to open the file transfer buffer. You have to accept the transfer from that screen.
